tools/depends/target/curl/Makefile

```diff
@@ -22,7 +22,7 @@ $(TARBALLS_LOCATION)/$(ARCHIVE):
 $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM)/*; mkdir -p $(PLATFORM)
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
-ifeq (darwin, $(findstring darwin, $(HOST)))
+ifeq ($(OS), darwin_embedded)
```

Comment:
Isn't this only required for iphoneos?

Reply:
Ya, I spotted that in the samba-gplv3 dependency but was not sure if it was the same. I guess it must be. Will change shortly.

Reply:
Done